nodes/clusters/nodes-cluster-resource-configure.adoc

```diff
@@ -26,6 +26,11 @@ managing application memory by:
 // assemblies.
 
 include::modules/nodes-cluster-resource-configure-about.adoc[leveloffset=+1]
+[discrete]
+[id="additional-resources_understanding-compute-resources"]
+== Additional resources
```

Comment:
Actually, if you're trying to make it look like an additional resources section of the module, then we use the following syntax:

```
.Additional resources
```